### DISCUSSION TITLE
pnpm and output standalone return EPERM: operation not permitted, symlink

### DISCUSSION THREAD

**Discussion by Aaqu** [vercel/next.js#52244] [Help] [answered]
### Summary

If I use next config `output: 'standalone'` with npm (package-lock.json) next build properly, pnpm (pnpm-lock.yaml) in build rerurn `EPERM: operation not permitted, symlink ...`. It it's normal?

![image](https://github.com/vercel/next.js/assets/49607785/51695858-987c-4a3f-8791-9dc38f3dd063)

**Comment by Aaqu** [comment]
now i think is pnpm problem

> ### Summary
> If I use next config `output: 'standalone'` with npm (package-lock.json) next build properly, pnpm (pnpm-lock.yaml) in build rerurn `EPERM: operation not permitted, symlink ...`. It it's normal?
> 
> ![image](https://user-images.githubusercontent.com/49607785/251110617-51695858-987c-4a3f-8791-9dc38f3dd063.png)
> 
> ### Additional information
> _No response_
> 
> ### Example
> _No response_

**Reply by Aaqu** [reply]
If I understand correctly windows, `symlinks` need admistrator permission.
Resolve for pnpm: 
creatre file `.npmrc` add `symlink=false`
remove `node_modules` and `.next` folders and run `pnpm install` 
now **build works**!


EDIT : in new project not work

**Reply by bangonkali** [reply]
I have the same error but encountered only on Windows. Docker based builds on CI/CD platform and Dockerfile execution using node:18-alpine do not encounter same issue.

**Comment by IvanTsxx** [comment, marked as answer]
You need to enable Windows Developer Mode so that Preconstruct can create symlinks - https://www.howtogeek.com/292914/what-is-developer-mode-in-windows-10/

This works for me 🫡

**Reply by CaseySMiller** [reply]
This!!! This is the answer.

**Reply by IvanTsxx** [reply]
Excellent! 🎊

**Reply by david-randoll** [reply]
This worked. Thanks

**Reply by liu245933567** [reply]
thank you!!!!

**Reply by pippinmole** [reply]
Where is this in the [installation instructions for windows](https://pnpm.io/installation#on-windows)? Sounds pretty important.

**Reply by princechrix** [reply]
I just turned on the Developer mode and it worked. thank you!

**Reply by IvanTsxx** [reply]
> Acabo de activar el modo Desarrollador y funcionó. ¡Gracias!

Excellent! 🎊

**Reply by zezo-developers** [reply]
thank u

**Comment by Eacolt** [comment]
I have encountered no difficulties using Nuxt, but I have faced numerous challenges when using Next.js. Next.js has caused me a great deal of uncertainty.

**Reply by SuperJakov** [reply]
This issue is related to pnpm causing issues.
